### PR TITLE
Make edx_django_service CONN_MAX_AGE configurable via variable for ea…

### PR DESCRIPTION
**Pull request by morenol**
The default value of edx_django_service_default_db_conn_max_age is 60. With this PR we can use differents values for each edx_django_service

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-3930](https://openedx.atlassian.net/browse/OSPR-3930) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by natabene** [comment]
@morenol Thank you for your contribution. @fredsmith Can you give this a look?

File: playbooks/roles/veda_web_frontend/meta/main.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ dependencies:
     edx_django_service_with_rendered_config_default_db_name: '{{ VEDA_WEB_FRONTEND_DEFAULT_DB_NAME }}'
     edx_django_service_with_rendered_config_db_user: '{{ VEDA_WEB_FRONTEND_MYSQL_USER }}'
     edx_django_service_with_rendered_config_db_password: '{{ VEDA_WEB_FRONTEND_MYSQL_PASSWORD }}'
+    edx_django_service_default_db_conn_max_age: '{{ VEDA_WEB_FRONTEND_MYSQLE_CONN_MAX_AGE }}'
```
Comment (felipemontoya) [inline playbooks/roles/veda_web_frontend/meta/main.yml]:
```suggestion
    edx_django_service_default_db_conn_max_age: '{{ VEDA_WEB_FRONTEND_MYSQL_CONN_MAX_AGE }}'
```

**Review by fredsmith — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@morenol 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/configuration&entry.867055334=https://github.com/edx/configuration/pull/5527&entry.1484655318=https://github.com/morenol&entry.752974735=2019-11-20+18:03&entry.1917517419=2020-01-24+19:13&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**